src/azure-cli/azure/cli/command_modules/acr/tests/latest/test_split_registry_and_image.py

```diff
@@ -0,0 +1,58 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import unittest
+import importlib
+
+from knack.util import CLIError
+
+acr_import = importlib.import_module('azure.cli.command_modules.acr.import')
```

Comment:
Did you have a particular reason for doing this?  Was it for clarity's sake

Regular import seems more straightforward to me... but you probably have your reasons
```suggestion
from azure.cli.command_modules.acr.import import _split_registry_and_image
```

Reply:
It was because the "import" is a reserved keyword and there would be syntax error.
![image](https://user-images.githubusercontent.com/8113721/76137455-17365480-6078-11ea-9bc7-872fa2c9f94b.png)

Reply:
ah I see! This makes sense. Maybe we should rename the package in another pr...